src/bin/contribute.rs

```diff
@@ -317,26 +315,24 @@ impl Contribute {
             }
         });
         // Force an update every 5 minutes.
-        let cloned2 = self.clone();
+        let cloned_for_update = self.clone();
         tokio::spawn(async move {
             loop {
-                match cloned2.get_locked_chunk_info().await {
+                match cloned_for_update.get_locked_chunk_info().await {
                     Err(err) => {
                         warn!("Cannot get locked chunks {}", err);
                     }
                     Ok(ceremony) => {
                         let mut found: Vec<String> = vec![];
+                        let v = cloned_for_update.get_participant_locked_chunk_ids();
```

Comment:
Nit: how about returning an error from the method and ignoring it/logging it here?